src/test/java/net/minestom/server/inventory/click/integration/HeldClickIntegrationTest.java

```diff
@@ -96,11 +105,12 @@ public void heldExternal(Env env) {
         var listener = env.listen(InventoryPreClickEvent.class);
         inventory.setItemStack(1, ItemStack.of(Material.DIAMOND));
         inventory.setItemStack(2, ItemStack.of(Material.GOLD_INGOT));
+        inventory.setItemStack(3, ItemStack.of(Material.EGG));
         inventory.setItemStack(4, ItemStack.of(Material.DIAMOND));
         // Empty
         {
             listener.followup(event -> {
-                assertEquals(inventory, event.getInventory());
+                if (event.getInventory() != null) assertEquals(inventory, event.getInventory());
```

Comment:
Note: This is needed because `InventoryClickProcessor#changeHeld` will fire the event twice, once using the player's inventory resulting in `getInventory` returning null.